scripts/Projects.psm1

```diff
@@ -11,7 +11,7 @@
 $Frameworks = @("net472")
 
 $Projects = @{}
-$Projects.Vsix = @("Sarif.Viewer.VisualStudio")
+$Projects.Vsix = @("Sarif.Viewer.VisualStudio", "Sarif.Sarifer")
```

Comment:
>Sarifer [](start = 55, length = 7)

Gabe, this is Michael's (possibly whimsical) suggestion for the VSIX names. "Sarifer" presumably because it makes SARIF, and it's the name of a beetle so you have a ready-made logo ;-)